New Page: Hallucination Detection Agent + Tutorial

DrZuzzjen opened the pull request:
### Description

**Hallucination Detection Agent page:**
-Starting guide
-Question - answer
-Chat completion

**New tutorial:**
Hallucination Dectection Agent Tutorial

![image](https://github.com/user-attachments/assets/3a6fe9e2-bf86-47e9-8be5-357527819a04)

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `kluster-mkdocs` repo

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+---
+title: Chat Completion Hallucination Detection
+description: Learn how to use the kluster.ai Hallucination Detection Agent to validate responses in full chat conversations.
```
```suggestion
description: Learn how to use the kluster.ai hallucination detection agent to validate responses in full chat conversations.
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+description: Learn how to use the kluster.ai Hallucination Detection Agent to validate responses in full chat conversations.
+---
+
+# Chat Completion
```
```suggestion
# Chat completion
```

albertov19 replied:
This might change depending on how we rename the page. Maybe to "Via chat completion API"

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+
+# Chat Completion
+
+The Chat Completion method allows you to validate responses in full conversation histories using the same format as the standard chat completions API. This approach enables detection of hallucinations within the complete context of a conversation.
```
```suggestion
You can perform hallucination detection within the complete context of a conversation via the chat completion method, as it allows you to validate responses in full conversation histories using the same format as the standard chat completions API. 
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+
+Before getting started with hallucination detection, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
```
```suggestion
--8<-- 'text/kluster-api-onboarding.md'
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
+
+## How it works
+
+The service evaluates the truthfulness of responses within a conversation by:
+
+1. Analyzing the entire conversation history, including system instructions.
+2. Examining the assistant's responses within context.
+3. Determining if the responses contain hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination and the search results used for fact-checking.
```
I would put this in the introduction! :) and then at the end of the intro I would put something like:

"This page shows you how you can perform hallucination detection using the chat completions endpoint via the kluster.ai API."

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+
+The agent operates in two distinct modes depending on whether you provide context with your request:
+
+- **Dedicated Endpoint** - use the API directly without specifying a model.
```
```suggestion
- **Dedicated endpoint**: Use the API directly without specifying a model.
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+The agent operates in two distinct modes depending on whether you provide context with your request:
+
+- **Dedicated Endpoint** - use the API directly without specifying a model.
+- **OpenAI SDK** - select the hallucination-detection model and use the SDK in a familiar way.
```
```suggestion
- **OpenAI SDK**: Select the hallucination-detection model and use the SDK in a familiar way.
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+
+### Dedicated endpoint
+
+By using the `/v1/judges/detect-hallucination` endpoint to check whether an assistant's answer is hallucination.
```
This page shold be all about chat completion right? Why cover this here and not in the question/answers?

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+
+## Next steps
+
+- Learn how to use [Question/Answer Hallucination Detection](/get-started/hallucination-agent/question-answer/){target=_self} for simpler verification scenarios
```
```suggestion
- Learn how to use [Question/answer hallucination detection](/get-started/hallucination-agent/question-answer/){target=_self} for simpler verification scenarios
```

albertov19 commented on get-started/hallucination-agent/chat-completion.md:
```diff
@@ -0,0 +1,124 @@
+## Next steps
+
+- Learn how to use [Question/Answer Hallucination Detection](/get-started/hallucination-agent/question-answer/){target=_self} for simpler verification scenarios
+- Explore [Examples](/get-started/hallucination-agent/examples/){target=_self} of hallucination detection in real-world scenarios
```
```suggestion
- Explore [examples](/get-started/hallucination-agent/examples/){target=_self} of hallucination detection in real-world scenarios
```

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+---
+title: Question - answer 
+description: Learn how to use the kluster.ai Hallucination Detection API to validate the truthfulness of answers to questions.
```
```suggestion
description: Learn how to use the kluster.ai hallucination detection API to validate the truthfulness of answers to questions.
```

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
```
```suggestion
--8<-- 'text/kluster-api-onboarding.md'
```

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+
+## How it works
+
+The service evaluates the truthfulness of an answer to a question by:
+
+1. Analyzing the original question or prompt.
+2. Examining the provided answer.
+3. Determining if the answer contains hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination as well as the search results used for fact-checking.
```
If this is the same text as in the other page I would move this to the overview section.

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+3. Determining if the answer contains hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination as well as the search results used for fact-checking.
+
+### Parameters
+
+| Parameter | Type | Required | Description |
+| :---: | :---: | :---: | :---: |
+| `prompt` | string | Yes | The question asked or instruction given. |
+| `output` | string | Yes | The answer to verify for hallucinations. |
+| `context` | string | No | Optional reference material to validate against. |
+| `return_search_results` | boolean | No | Whether to include search results (default: false). |
```
Before mentioning parameters we should mention the endpoint right?

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+
+The agent operates in two distinct modes depending on whether you provide context with your request:
+
+- **Fact-checking mode** - when no context is provided, the agent verifies answers against general knowledge and external sources.
```
```suggestion
- **Fact-checking mode**: When no context is provided, the agent verifies answers against general knowledge and external sources.
```

albertov19 replied:
Maybe briefly mention how is context provided or not provided? so that is dead obvious

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+The agent operates in two distinct modes depending on whether you provide context with your request:
+
+- **Fact-checking mode** - when no context is provided, the agent verifies answers against general knowledge and external sources.
+- **Context validation mode** - when context is provided, the agent only validates answers against the specified context.
```
```suggestion
- **Context validation mode**: When context is provided, the agent only validates answers against the specified context.
```

albertov19 commented on get-started/hallucination-agent/question-answer.md:
```diff
@@ -0,0 +1,208 @@
+        "Authorization": f"Bearer {api_key}",
+        "Content-Type": "application/json"
+    }
+    payload = {
```
minor nitpick but try to use standardized snippet variable names etc. Check the quickstart snippets to use as reference, this way, readers get used to variable names and understand what is this here etc

DrZuzzjen commented:
![image](https://github.com/user-attachments/assets/4ece7b1b-451b-42ec-b648-6c71e812fbef)

Update:

- Rebranding: kluster verify / Reliability check.
- Refactored folders/links to match new service structure. 
- Workflows  + screenshots added
- DISABLED MCP integration section (will later be re-defined)
- Replaced all traces of "fact", "fact-checking",etc.
- Updated notebook with the same above refs

@albertov19  @Telucero: Your comments have been addressed as well.   

Please review:
- chat_completion.md
- question-answer.md
- starting-guide.md
- workflow-integrations.md
- tutorials/klusterai-api/reliability-check.ipynb

themacexpert commented on get-started/verify/reliability/overview.md:
```diff
@@ -0,0 +1,107 @@
+
+    Verify the reliability and accuracy of an answer to a specific question via a dedicated API endpoint.
+
+    [:octicons-arrow-right-24: Visit the guide](/get-started/verify/reliability/question-answer/){target=\_self}
```
```suggestion
    [:octicons-arrow-right-24: Visit the guide](/get-started/verify/reliability/dedicated-api/){target=\_self}
```

themacexpert commented on get-started/verify/reliability/overview.md:
```diff
@@ -0,0 +1,107 @@
+
+    [:octicons-arrow-right-24: Visit the guide](/get-started/verify/reliability/question-answer/){target=\_self}
+
+-   <span class="badge guide">Guide</span> __Chat completion endpoint
```
```suggestion
-   <span class="badge guide">Guide</span> Chat completion endpoint
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
````diff
@@ -0,0 +1,139 @@
+
+1. **Build the server**:
+   ```bash
+   # TODO: Define MCP repository URL
````
```suggestion
   # Define MCP repository URL
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
````diff
@@ -0,0 +1,139 @@
+   ```bash
+   # TODO: Define MCP repository URL
+   git clone https://github.com/kluster-ai/hallucination-mcp-server
+   cd verify-mcp-server  # TODO: Update folder name when repo is renamed
````
```suggestion
   cd verify-mcp-server  
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
````diff
@@ -0,0 +1,139 @@
+```
+
+!!! info "Workflow Integrations"
+    For platforms like Dify, n8n, and Zapier that use HTTP requests, see [Workflow Integrations](/get-started/verify/reliability/workflow-integrations/){target=\_self}
````
```suggestion
    For platforms like Dify, n8n, and Zapier that use HTTP requests, see [Workflow Integrations](/get-started/verify/reliability/workflow-integrations/){target=\_blank}
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
```diff
@@ -0,0 +1,139 @@
+
+## Additional Resources
+
+- **Quick Start** - Get running in [5 minutes](/get-started/verify/reliability/mcp/quick-start/){target=\_self}
```
```suggestion
- **Quick Start** - Get running in [5 minutes](/get-started/verify/reliability/mcp/quick-start/){target=\_blank}
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
```diff
@@ -0,0 +1,139 @@
+## Additional Resources
+
+- **Quick Start** - Get running in [5 minutes](/get-started/verify/reliability/mcp/quick-start/){target=\_self}
+- **Tools Reference** - Detailed [parameter documentation](/get-started/verify/reliability/mcp/tools/){target=\_self}
```
```suggestion
- **Tools Reference** - Detailed [parameter documentation](/get-started/verify/reliability/mcp/tools/){target=\_blank}
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
```diff
@@ -0,0 +1,139 @@
+
+- **Quick Start** - Get running in [5 minutes](/get-started/verify/reliability/mcp/quick-start/){target=\_self}
+- **Tools Reference** - Detailed [parameter documentation](/get-started/verify/reliability/mcp/tools/){target=\_self}
+- **MCP Integration** - Return to the [main guide](/get-started/verify/reliability/mcp/){target=\_self}
```
```suggestion
- **MCP Integration** - Return to the [main guide](/get-started/verify/reliability/mcp/){target=\_blank}
```

themacexpert commented on get-started/verify/reliability/mcp/clients.md:
```diff
@@ -0,0 +1,139 @@
+- **Quick Start** - Get running in [5 minutes](/get-started/verify/reliability/mcp/quick-start/){target=\_self}
+- **Tools Reference** - Detailed [parameter documentation](/get-started/verify/reliability/mcp/tools/){target=\_self}
+- **MCP Integration** - Return to the [main guide](/get-started/verify/reliability/mcp/){target=\_self}
+- **Workflow Integrations** - HTTP-based [workflow templates](/get-started/verify/reliability/workflow-integrations/){target=\_self}
```
```suggestion
- **Workflow Integrations** - HTTP-based [workflow templates](/get-started/verify/reliability/workflow-integrations/){target=\_blank}
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
Where is this page in the navigation? I couldn't find it without going directly to it: /get-started/verify/reliability/mcp/

albertov19 replied:
on purpose. I'll remove it from this PR and move it to another PR tbh

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+
+# Integration
+
+LLM applications sometimes generate convincing but false information. This breaks user trust and creates liability. The **kluster verify MCP server** solves this by adding real-time fact-checking to any MCP-compatible environment.
```
```suggestion
LLM applications sometimes generate convincing but false information. This breaks user trust and creates liability. The kluster verify MCP server solves this by adding real-time fact-checking to any MCP-compatible environment.
```

themacexpert commented on get-started/verify/reliability/chat-completion.md:
```diff
@@ -0,0 +1,117 @@
+Before getting started with reliability verification, ensure the following requirements are met:
+
+--8<-- 'text/kluster-api-onboarding.md'
+- **A virtual Python environment** - (optional) recommended for developers using Python. It helps isolate Python installations in a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/){target=\_blank} to reduce the risk of environment or package conflicts between your projects
```
```suggestion
- **A virtual Python environment**: (Optional) Recommended for developers using Python. It helps isolate Python installations in a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/){target=\_blank} to reduce the risk of environment or package conflicts between your projects
```

themacexpert commented on get-started/verify/reliability/chat-completion.md:
```diff
@@ -0,0 +1,117 @@
+
+--8<-- 'text/kluster-api-onboarding.md'
+- **A virtual Python environment** - (optional) recommended for developers using Python. It helps isolate Python installations in a [virtual environment](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/){target=\_blank} to reduce the risk of environment or package conflicts between your projects
+- **Required Python libraries** - install the following Python libraries:
```
```suggestion
- **Required Python libraries**: Install the following Python libraries:
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+
+- **Dedicated reliability endpoint** - Learn how to [verify individual question-answer pairs](/get-started/verify/reliability/dedicated-api/){target=\_self}
+- **Chat Completion verification** - Discover how to [validate responses in conversations](/get-started/verify/reliability/chat-completion/){target=\_self}
+- **Practical examples** - Explore our [Tutorials](/tutorials/klusterai-api/reliability-check){target=\_blank} for kluster verify
```
```suggestion
- **Practical examples**: Explore our [Tutorials](/tutorials/klusterai-api/reliability-check){target=\_blank} for kluster verify.
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+- **Dedicated reliability endpoint** - Learn how to [verify individual question-answer pairs](/get-started/verify/reliability/dedicated-api/){target=\_self}
+- **Chat Completion verification** - Discover how to [validate responses in conversations](/get-started/verify/reliability/chat-completion/){target=\_self}
+- **Practical examples** - Explore our [Tutorials](/tutorials/klusterai-api/reliability-check){target=\_blank} for kluster verify
+- **API reference** - Review the complete [API documentation](/api-reference/reference/){target=\_blank} for all reliability verification endpoints
```
```suggestion
- **API reference**: Review the complete [API documentation](/api-reference/reference/){target=\_blank} for all reliability verification endpoints.
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+
+The server integrates with your applications by:
+
+1. **Exposing tools** through the standard MCP protocol
```
```suggestion
1. Exposing tools through the standard MCP protocol
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+The server integrates with your applications by:
+
+1. **Exposing tools** through the standard MCP protocol
+2. **Processing claims** using kluster.ai's verification API  
```
```suggestion
2. Processing claims using kluster.ai's verification API  
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+
+1. **Exposing tools** through the standard MCP protocol
+2. **Processing claims** using kluster.ai's verification API  
+3. **Returning results** with explanations and source citations
```
```suggestion
3. Returning results with explanations and source citations
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+1. **Exposing tools** through the standard MCP protocol
+2. **Processing claims** using kluster.ai's verification API  
+3. **Returning results** with explanations and source citations
+4. **Working everywhere** MCP is supported
```
```suggestion
4. Working everywhere MCP is supported
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+Before getting started with MCP integration, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=\_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=\_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=\_blank} guide
```
```suggestion
```

themacexpert commented on get-started/verify/reliability/mcp/index.md:
```diff
@@ -0,0 +1,93 @@
+
+Before getting started with MCP integration, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=\_blank} if you do not have one
```
```suggestion
--8<-- 'text/kluster-api-onboarding.md'
```

themacexpert commented on get-started/verify/reliability/mcp/quick-start.md:
```diff
@@ -0,0 +1,99 @@
+Before getting started with MCP integration, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=\_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=\_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=\_blank} guide
```
```suggestion
```

themacexpert commented on get-started/verify/reliability/mcp/quick-start.md:
```diff
@@ -0,0 +1,99 @@
+
+Before getting started with MCP integration, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=\_blank} if you do not have one
```
```suggestion
--8<-- 'text/kluster-api-onboarding.md'
```

themacexpert commented on get-started/verify/reliability/mcp/tools.md:
```diff
@@ -0,0 +1,192 @@
+
+### Use Cases
+
+- **Legal documents** - Contract terms, policy interpretation
```
```suggestion
- **Legal documents**: Contract terms, policy interpretation
```

themacexpert commented on get-started/verify/reliability/mcp/tools.md:
```diff
@@ -0,0 +1,192 @@
+### Use Cases
+
+- **Legal documents** - Contract terms, policy interpretation
+- **Research papers** - Data claims, methodology verification  
```
```suggestion
- **Research papers**: Data claims, methodology verification  
```

themacexpert commented on get-started/verify/reliability/mcp/tools.md:
```diff
@@ -0,0 +1,192 @@
+
+- **Legal documents** - Contract terms, policy interpretation
+- **Research papers** - Data claims, methodology verification  
+- **Financial reports** - Revenue figures, growth claims
```
```suggestion
- **Financial reports**: - Revenue figures, growth claims
```

themacexpert commented on get-started/verify/reliability/mcp/tools.md:
```diff
@@ -0,0 +1,192 @@
+- **Legal documents** - Contract terms, policy interpretation
+- **Research papers** - Data claims, methodology verification  
+- **Financial reports** - Revenue figures, growth claims
+- **Technical specifications** - Feature descriptions, requirements
```
```suggestion
- **Technical specifications**: - Feature descriptions, requirements
```

themacexpert commented on get-started/verify/reliability/mcp/tools.md:
```diff
@@ -0,0 +1,192 @@
+- **Quick Start** - Get up and running with the [5-minute setup guide](/get-started/hallucination-agent/mcp-quick-start/){target=\_self}
+- **Client Setup** - Configure [different MCP clients](/get-started/hallucination-agent/mcp-clients/){target=\_self}  
+- **MCP Integration** - Return to the [main guide](/get-started/hallucination-agent/mcp/){target=\_self}
+- **API Reference** - Complete [API documentation](/api-reference/reference/){target=\_blank}
```
```suggestion
- **API Reference**: Complete [API documentation](/api-reference/reference/){target=\_blank}.
```

DrZuzzjen replied:
Addressed

themacexpert commented on get-started/verify/reliability/overview.md:
```diff
@@ -0,0 +1,107 @@
+## Additional resources
+
+- **Workflow Integrations**: Download [ready-to-use workflows for Dify, n8n](/get-started/verify/reliability/workflow-integrations/){target=\\_self}
+- **Tutorial**: Explore the [kluster verify tutorial](/tutorials/klusterai-api/reliability-check){target=\_blank} with code examples
```
```suggestion
- **Tutorial**: Explore the [kluster verify tutorial](/tutorials/klusterai-api/reliability-check){target=\_blank} with code examples.
```

themacexpert commented on get-started/verify/reliability/workflow-integrations.md:
```diff
@@ -0,0 +1,91 @@
+
+Ready to build more reliable AI applications?
+
+- **Explore the API**: Check the [complete API reference](/api-reference/reference/){target=\_blank} for advanced configuration options
```
```suggestion
- **Explore the API**: Check the [complete API reference](/api-reference/reference/){target=\_blank} for advanced configuration options.
```

themacexpert commented on get-started/verify/reliability/workflow-integrations.md:
```diff
@@ -0,0 +1,91 @@
+Ready to build more reliable AI applications?
+
+- **Explore the API**: Check the [complete API reference](/api-reference/reference/){target=\_blank} for advanced configuration options
+- **Learn verification methods**: Dive into the [dedicated reliability endpoint](/get-started/verify/reliability/dedicated-api/){target=\_blank} for detailed implementation patterns
```
```suggestion
- **Learn verification methods**: Dive into the [dedicated reliability endpoint](/get-started/verify/reliability/dedicated-api/){target=\_blank} for detailed implementation patterns.
```

themacexpert commented on get-started/verify/reliability/workflow-integrations.md:
```diff
@@ -0,0 +1,91 @@
+
+- **Explore the API**: Check the [complete API reference](/api-reference/reference/){target=\_blank} for advanced configuration options
+- **Learn verification methods**: Dive into the [dedicated reliability endpoint](/get-started/verify/reliability/dedicated-api/){target=\_blank} for detailed implementation patterns
+- **Try the tutorial**: Follow the [hands-on reliability check tutorial](/tutorials/klusterai-api/reliability-check/){target=\_blank} with code examples
```
```suggestion
- **Try the tutorial**: Follow the [hands-on reliability check tutorial](/tutorials/klusterai-api/reliability-check/){target=\_blank} with code examples.
```

DrZuzzjen replied:
Addressed

themacexpert reviewed (commented):
Nice PR! This is a big one! 
I'm commenting here rather than requesting changes so my review doesn't block the PR approval process.

themacexpert commented:
Tutorial is looking great! I ran thru it in Jupyter notebook no problems

DrZuzzjen commented on get-started/.pages:
```diff
@@ -6,3 +6,4 @@ nav:
   - start-building
   - 'OpenAI compatibility': openai-compatibility.md
   - 'Integrations': integrations
+  - 'kluster.ai verify': verify
```
I agree, looks repetitive to say the .ai part again. But the name could change soon so lets keep it like this for the moment.

albertov19 commented on get-started/verify/reliability/mcp/clients.md:
```diff
@@ -0,0 +1,139 @@
+---
+title: Clients
+description: Configure kluster verify's MCP server with Claude Desktop, VS Code, and other MCP-compatible platforms.
```
This page is hiddlen as we are not releasing this yet

albertov19 commented on get-started/verify/reliability/workflow-integrations.md:
```diff
@@ -0,0 +1,91 @@
+
+The workflow includes pre-configured HTTP nodes that connect to the `/v1/verify/reliability` endpoint, handle request/response formatting, and parse verification results. Connect your data sources and configure output routing as needed.
+
+[Download n8n Workflow](workflows/n8n_workflow.json){target=\_blank .md-button}
```
Sounds good - Please let's open an issue for this
